Makefile

```
@@ -232,15 +231,19 @@ test-upgrade-e2e: kind-cluster run-latest-release image-registry pre-upgrade-set
 e2e-coverage:
 	COVERAGE_OUTPUT=./coverage/e2e.out ./hack/test/e2e-coverage.sh
 
-.PHONY: kind-load
-kind-load: $(KIND) #EXHELP Loads the currently constructed image onto the cluster.
+kind-load: $(KIND) build-catalogd #EXHELP Loads the currently constructed images into the KIND cluster.
 	$(CONTAINER_RUNTIME) save $(IMG) | $(KIND) load image-archive /dev/stdin --name $(KIND_CLUSTER_NAME)
+	$(CONTAINER_RUNTIME) save quay.io/operator-framework/catalogd:devel | $(KIND) load image-archive /dev/stdin --name $(KIND_CLUSTER_NAME)
 
 .PHONY: kind-deploy
-kind-deploy: export MANIFEST="./operator-controller.yaml"
-kind-deploy: manifests $(KUSTOMIZE) #EXHELP Install controller and dependencies onto the kind cluster.
+kind-deploy: export MANIFEST := ./operator-controller.yaml
+kind-deploy: manifests $(KUSTOMIZE) #EXHELP Install controller and dependencies, including catalogd, onto the kind cluster.
 	$(KUSTOMIZE) build $(KUSTOMIZE_BUILD_DIR) > operator-controller.yaml
-	envsubst '$$CATALOGD_VERSION,$$CERT_MGR_VERSION,$$INSTALL_DEFAULT_CATALOGS,$$MANIFEST' < scripts/install.tpl.sh | bash -s
+	cd catalogd && $(KUSTOMIZE) build config/overlays/cert-manager > catalogd.yaml
+	envsubst '$$CERT_MGR_VERSION,$$INSTALL_DEFAULT_CATALOGS,$$MANIFEST' < scripts/install.tpl.sh > install.sh
+	bash install.sh \
+		--catalogd-manifest=./catalogd/catalogd.yaml \
+		--default-catalogs-manifest=./catalogd/config/base/default/clustercatalogs/default-catalogs.yaml
```

Do we need a separate `catalogd.yaml` manifest? I wonder if we could append the catalogd manifests into `operator-controller.yaml`? (Maybe rename to `olmv1.yaml`?)

If we did that, you'd no longer need the special handling in the install script for the catalogd manifest, and it would be one fewer release artifact to manage.

I'm thinking something like this:
```suggestion
	$(KUSTOMIZE) build $(KUSTOMIZE_BUILD_DIR) && $(KUSTOMIZE) build $(CATALOGD_KUSTOMIZE_BUILD_DIR) > operator-controller.yaml
 	envsubst '$$CERT_MGR_VERSION,$$INSTALL_DEFAULT_CATALOGS,$$MANIFEST' < scripts/install.tpl.sh | bash -s
```

WDYT?

---

I tried to:

> Do we need a separate catalogd.yaml manifest? I wonder if we could append the catalogd manifests into operator-controller.yaml? (Maybe rename to olmv1.yaml?)

But the problem is that it fails because it is missing the CRDs
IHMO: We need to decide whether to have two managers or combine all in the controller-operator to make things easier and then combine the manifests.

---

I could do that..  but still needing more follow ups now to fix it.